examples/simple/src/users/UserCreate.tsx

```diff
@@ -70,7 +76,7 @@ const UserCreate = () => {
                         validate={[required(), isValidName, unique()]}
                     />
                 </TabbedForm.Tab>
-                {permissions === 'admin' && (
+                {canManageUsers ? (
```

Comment:
Could not use `<CanAccess>` here because `<TabbedForm>` still scans its children, so having `<CanAccess>` as one of its child couldn't work.